app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CustomerItemsController < ApplicationController
+  def create
+    CustomerItems.create(customer_id: params[:customer_id], item_id: params[:item_id])
```

Comment:
Awesome set up here! This is the most restful option: putting this functionality in a joins controller `create` rather than a `create` in the CustomersController. Nice work!